relation_engine/batchload/test/load_utils_test.py

```diff
@@ -1,145 +0,0 @@
-from copy import deepcopy
-from io import StringIO
-from relation_engine.batchload.load_utils import process_nodes, process_edge, process_edges
-
-
-_MAX_ADB_INT = 9007199254740991
-
-
-def test_process_nodes_noop():
-    outfile = StringIO()
-    process_nodes([], 'ver', 1, outfile)
-    assert outfile.getvalue() == ''
-
-
-def test_process_nodes():
```

Comment:
For the deleted test files, are there equivalent tests for the delta loader where needed? delta_load_integration_test.py seems to be it but just thought I'd ask to double check.

Reply:
yes, the big green files in the sunburst are the 2 delta loader core code files https://app.codecov.io/gh/kbase/relation_engine_importers